sql-basics/scripts.sql

```diff
@@ -0,0 +1,38 @@
+Basic selects:
+
+1. SELECT * FROM world.city; 
+2. SELECT name FROM city;
+3. SELECT name FROM city WHERE CountryCode = "HUN";
+4. SELECT DISTINCT CountryCode FROM city ORDER BY CountryCode ASC;
+5. SELECT DISTINCT CountryCode FROM city ORDER BY CountryCode DESC;
+6. SELECT name FROM city WHERE CountryCode = "ITA";
+7. SELECT name FROM city WHERE CountryCode = "AFG";
+8. SELECT name, population FROM city WHERE CountryCode = "USA";
+9. SELECT name, population FROM city WHERE CountryCode = "IND" ORDER BY population ASC;
+10. SELECT name, population FROM city WHERE CountryCode = "IND" ORDER BY population DESC;
+11. SELECT * FROM City WHERE Name LIKE "New%";
+12. SELECT name FROM city WHERE LENGTH(name) < 7;
```

Comment:
remember the difference between `LENGTH` and `CHAR_LENGTH` - also, please research a bit in which type cities this difference could distort the result